### ISSUE TITLE
Error with ODE system in post processing in modelsearch

### ISSUE THREAD

**Issue by stellabelin** [pharmpy/pharmpy#696]

**Comment by rikardn** [comment]
In post processing:

```
'AttributeError("\'ExplicitODESystem\' object has no attribute \'amounts\'")'
```

for models with `$DES`

**Comment by rikardn** [comment]
Fixed with 9ff2d2e3c88ee509abeb5bf17ca1a23fe9d02866 and bfc01fe17b8879e04c38955c15bf3448d16b601d
